zipkin-server/README.md

```diff
@@ -224,6 +224,14 @@ to prevent excessive load, service and span name queries are limited by
 `QUERY_LOOKBACK`, which defaults to 24hrs (two daily buckets: one for
 today and one for yesterday)
 
+### HTTP Collector
+The HTTP collector is enabled by default. It accepts spans via `POST /api/v1/spans`. The HTTP 
+collector supports the following configuration:
+
+Property | Environment Variable | Description
+--- | --- | ---
+`zipkin.collector.http.enabled` | `HTTP_COLLECTOR_ENABLED` | `false` disables the HTTP collector. Defaults to `true`.
```

Comment:
As far as the naming or existence of the environment variable, I'm happy to leave it in or take it out (of the docs and [zipkin-server-shared.yml](zipkin-server/src/main/resources/zipkin-server-shared.yml)). The one argument I can think of in favor of keeping it is support for disabling the HTTP collector for users of the openzipkin/zipkin docker image. Either way won't affect my particular situation, because I am already configuring other aspects of zipkin-server via Java system properties.